README.md

```diff
@@ -35,62 +35,56 @@ For building documentation:
 ## Build and Install the SLEIGH Library
 
 ```sh
-# Clone this repository (CMake project for SLEIGH)
+# Clone this repository (CMake project for sleigh)
 git clone https://github.com/lifting-bits/sleigh.git
 cd sleigh
 
 # Configure CMake
 cmake -B build -S .
 
 # Build SLEIGH
-cmake --build build -j
+cmake --build build --parallel 8
 
 # Install SLEIGH
-cmake --install build --prefix <path where SLEIGH will install>
+cmake --install build --prefix ./install
 ```
 
 ### Note on Ghidra source code
 
 The Ghidra source code is not actually included in this git repo, and by default, CMake will automatically pull a stable version from the internet for you.
 
-Please see [`src/README.md`](./src/README.md) for more information on how to customize which Ghidra source code commit will be used/compiled, including specifying your own local copy of the Ghidra source.
+Please see [`src/README.md`](src/README.md) for more information on how to customize which Ghidra source code commit will be used/compiled, including specifying your own local copy of the Ghidra source.
 
 ## Packaging
 
-The CMake configuration also supports building packages for SLEIGH. If the `sleigh_ENABLE_PACKAGING` option is set during the configuration step, the build step will generate a tarball containing the SLEIGH installation. Additionally, the build will create an RPM package if it finds `rpm` in the `PATH` and/or a DEB package if it finds `dpkg` in the `PATH`.
+The CMake configuration also supports building packages for SLEIGH.
 
 For example:
 
 ```sh
-cmake -B build -S . \
-    -Dsleigh_ENABLE_PACKAGING=ON
-
-# Build SLEIGH
-cmake --build build -j
-
 # Package SLEIGH
 cmake --build build --target package
 ```
 
 ## API Usage
 
-An example program called `sleigh-lift` has been included to demonstrate how to use the SLEIGH API. It takes a hexadecimal string of bytes and either disassembles it or lifts it to p-code. The program can be invoked like so, where the `action` argument must be either `disassemble` or `pcode`:
+An example program called `sleighLift` has been included to demonstrate how to use the Sleigh API. It takes a hexadecimal string of bytes and can disassemble it or lift it to p-code. The program can be invoked as follows, where the `action` argument must be either `disassemble` or `pcode`:
 
 ```sh
-sleigh-lift [action] [sla_file] [bytes] [-a address] [-p root_sla_dir] [-s pspec_file]
+sleighLift [action] [sla_file] [bytes] [-a address] [-p root_sla_dir] [-s pspec_file]
```

Comment:
Why are we naming the tool like this? This doesn't match the convention of our other codebases or Sleigh itself.

Reply:
Fixed.